libsplinter/src/admin/service/event/store/diesel/models.rs

```diff
@@ -92,6 +92,7 @@ pub struct AdminEventProposedCircuitModel {
     pub comments: Option<String>,
     pub display_name: Option<String>,
     pub circuit_version: i32,
+    pub circuit_status: String,
```

Comment:
This should be an enum (see https://github.com/Cargill/splinter/blob/master/libsplinter/src/rest_api/auth/rbac/store/diesel/models.rs#L46-L89 for an example)

Reply:
Updated to be represented as an enum in the admin store, and event store